lib/tasks/upgrade.rake

```diff
@@ -0,0 +1,17 @@
+namespace :upgrade do
+  desc "Run sidekiq with old queues so it can finish deferred jobs"
+  task :run_legacy_queues do
+    system "bundle exec sidekiq --daemon -q maintenance -q dispatch -q delete_account -q http -q http_service"\
+           " -q export -q photos -q socket_webfinger -q mail -q receive_local -q receive -q receive_salmon"\
```

Comment:
I'm not sure if we need this rake-task anyway. It is useful for people running already on develop, but when people upgrading from 0.5.x to 0.6.0.0 want to run queued jobs, they need to start sidekiq again before git pull, wait, and kill after 5 minutes and then pull and do the rest of the upgrade (like on [0.4 -> 0.5, step 4](https://wiki.diasporafoundation.org/Updating#Updating_diaspora.2A_0.4_to_diaspora.2A_0.5)). But this rake-task is not useful for them.